src/uffs/uffs_tree.c

```diff
@@ -1082,6 +1085,9 @@ static URET _BuildTreeStepThree(uffs_Device *dev)
 	uffs_Perror(UFFS_MSG_NOISY, "build tree step three");
 
 	for (i = 0; i < DATA_NODE_ENTRY_LEN; i++) {
+              if (dev->OnBuildTreeStepThreeBlockProcess != NULL) {
+                      dev->OnBuildTreeStepThreeBlockProcess(i);
```

Comment:
`i` is a node entry index and it is different from block number

Reply:
fixed